src/Uno.UI/UI/Xaml/Controls/Border/Border.Android.cs

```diff
@@ -108,6 +107,9 @@ protected override void OnBackgroundChanged(DependencyPropertyChangedEventArgs e
 		{
 			UpdateBorder();
 		}
+
+		bool ICustomClippingElement.AllowClippingToLayoutSlot => !(Child is UIElement ue) || ue.RenderTransform == null;
+		bool ICustomClippingElement.ForceClippingToLayoutSlot => CornerRadius != CornerRadius.None;
```

Comment:
😯 that's a crazy magic unexpected behavior ... wondering however why the rules are not the same on all platforms ?

Reply:
It's because iOS is doing the clipping for rounded corners elsewhere, so the "clipping logic" must be deactivated for a control when rounded corners is activated on this platform.